src/features/offer/components/VenueSelectionList/VenueSelectionList.tsx

```diff
@@ -1,54 +1,74 @@
 import React, { useCallback } from 'react'
-import { View, ViewProps } from 'react-native'
-import styled from 'styled-components/native'
+import { FlatList, FlatListProps, View, ViewProps } from 'react-native'
+import styled, { useTheme } from 'styled-components/native'
 
 import { VenueSelectionListItem } from 'features/offer/components/VenueSelectionListItem/VenueSelectionListItem'
 import { VenueDetail } from 'features/offer/types'
-import { Li } from 'ui/components/Li'
-import { VerticalUl } from 'ui/components/Ul'
 import { getSpacing } from 'ui/theme'
 
 export type VenueListItem = VenueDetail & {
   offerId: number
 }
 
-export type VenueSelectionListProps = ViewProps & {
-  selectedItem?: number
-  onItemSelect: (itemOfferId: number) => void
-  items: VenueListItem[]
-}
+export type VenueSelectionListProps = ViewProps &
+  Pick<FlatListProps<VenueListItem>, 'onRefresh' | 'refreshing' | 'onEndReached' | 'onScroll'> & {
+    selectedItem?: number
+    onItemSelect: (itemOfferId: number) => void
+    items: VenueListItem[]
+  }
+
+const keyExtractor = (item: VenueListItem) => String(item.offerId)
 
 export function VenueSelectionList({
   items,
   selectedItem,
   onItemSelect,
+  onEndReached,
+  refreshing,
+  onRefresh,
+  onScroll,
   ...props
 }: VenueSelectionListProps) {
+  const { modal } = useTheme()
+
+  const handleOnSelect = useCallback(
+    (itemOfferId: number) => onItemSelect(itemOfferId),
+    [onItemSelect]
+  )
+
   const renderItem = useCallback(
-    (item: VenueListItem) => {
+    ({ item }: { item: VenueListItem }) => {
       return (
         <ItemWrapper>
           <VenueSelectionListItem
             {...item}
-            onSelect={() => onItemSelect(item.offerId)}
+            onSelect={() => handleOnSelect(item.offerId)}
             isSelected={selectedItem === item.offerId}
           />
         </ItemWrapper>
       )
     },
-    [onItemSelect, selectedItem]
+    [handleOnSelect, selectedItem]
   )
```

Comment:
Pas utile du coup ? Pourquoi tu ne laisses pas comme c'était ?

```suggestion
  const renderItem = useCallback(
    ({ item }: { item: VenueListItem }) => {
      return (
        <ItemWrapper>
          <VenueSelectionListItem
            {...item}
            onSelect={() => onItemSelect(item.offerId)}
            isSelected={selectedItem === item.offerId}
          />
        </ItemWrapper>
      )
    },
    [onItemSelect, selectedItem]
  )
```

Reply:
Yes effectivement j'ai eu un gros doute sur ce que j'ai fait. Du coup sorry pour le `useCallback` que tu voulais ici